### ISSUE TITLE
Does not work on ios react-native version 0.60.x

### ISSUE THREAD

**Issue by henry983** [januslo/react-native-bluetooth-escpos-printer#73]
I hope you will support react native 0. 60. x soon. I am looking forward to the update 0. 0.6 from you or manual installation instructions on ios manual.

**Comment by JuanCamilo0809** [comment]
I have the same problem

**Comment by JuanCamilo0809** [comment]
> I hope you will support react native 0. 60. x soon. I am looking forward to the update 0. 0.6 from you or manual installation instructions on ios manual.

Any solution?

**Comment by edritech93** [comment]
same for me, any solution ?

**Comment by JuanCamilo0809** [comment]
> same for me, any solution ?

You have to add de pod file manually. 
https://github.com/januslo/react-native-bluetooth-escpos-printer/pull/76

It´s working for me and i am using 0.61.2

**Comment by fabicsp** [comment]
I have tried to install it via npm, and in that case also got the same error.
When installed with:  **npm install https://github.com/januslo/react-native-bluetooth-escpos-printer.git --save** It worked.

**Comment by januslo** [comment]
Just merged, but not release yet.

发自我的iPhone

**Comment by jeafgilbert** [comment]
I'm also having problem running with ReactNative 0.6. I did fresh install project (without Expo), install this plugin, but BluetoothManager is always null. I have tried using `const { BluetoothManager } = NativeModuels ` but no luck. I think I miss something on cocoapods procedure but don't know exactly how. Could you please provide the steps? (cc: @JuanCamilo0809)

**Comment by afrakhan123** [comment]
this works in package.json :  "react-native-bluetooth-escpos-printer": "https://github.com/januslo/react-native-bluetooth-escpos-printer.git",

**Comment by jeafgilbert** [comment]
@afrakhan123 Yup, I installed plugin via github with "npm install https://github.com/januslo/react-native-bluetooth-escpos-printer.git --save" instead and it's working, but I got this further error on 'Scan' tapped:

<img width="364" alt="Screen Shot 2020-05-23 at 6 32 43 PM" src="https://user-images.githubusercontent.com/1263134/82729801-97d62a00-9d24-11ea-8835-adf1b8527ef8.png">

Please advice.

**Comment by jeafgilbert** [comment]
@JuanCamilo0809 Hi, have u experienced this issue?

> @afrakhan123 Yup, I installed plugin via github with "npm install https://github.com/januslo/react-native-bluetooth-escpos-printer.git --save" instead and it's working, but I got this further error on 'Scan' tapped:
> 
> <img alt="Screen Shot 2020-05-23 at 6 32 43 PM" width="364" src="https://user-images.githubusercontent.com/1263134/82729801-97d62a00-9d24-11ea-8835-adf1b8527ef8.png">
> 
> Please advice.

**Comment by jeafgilbert** [comment]
Never mind. In the end I reinstalled project with React Native v0.59.8. Great plugin though, thanks.

**Comment by JuanCamilo0809** [comment]
> @JuanCamilo0809 Hi, have u experienced this issue?
> 
> > @afrakhan123 Yup, I installed plugin via github with "npm install https://github.com/januslo/react-native-bluetooth-escpos-printer.git --save" instead and it's working, but I got this further error on 'Scan' tapped:
> > <img alt="Screen Shot 2020-05-23 at 6 32 43 PM" width="364" src="https://user-images.githubusercontent.com/1263134/82729801-97d62a00-9d24-11ea-8835-adf1b8527ef8.png">
> > Please advice.

Yes, but it was because the bluetooth is off, try On and it´s ok.
